src/screenComponents/selfDestructIndicator.cpp

```diff
@@ -29,16 +29,16 @@ void GuiSelfDestructIndicator::onDraw(sf::RenderTarget& window)
             for(int n=0; n<PlayerSpaceship::max_self_destruct_codes; n++)
                 if (!my_spaceship->self_destruct_code_confirmed[n])
                     todo++;
-            label->setText("Waiting for autorization input: " + string(todo) + " left");
+            label->setText(tr("Waiting for autorization input: {codes} left").format({{"codes", string(todo)}}));
         }else{
             if (my_spaceship->self_destruct_countdown <= 3.0f)
             {
-                label->setText("Have a nice day.");
+                label->setText(tr("Have a nice day."));
             }
             else
             {
                 char buffer[46];
-                snprintf(buffer, 46, "This ship will self-destruct in %.0f seconds.", my_spaceship->self_destruct_countdown);
+                snprintf(buffer, 46, tr("This ship will self-destruct in %.0f seconds."), my_spaceship->self_destruct_countdown);
```

Comment:
How to   %.0f

Reply:
Oof, this is old code :) let me see if I can fix this in a better way.